src/elements/callout/calloutDataTransformer.ts

```diff
@@ -38,14 +59,37 @@ export const transformElementOut: TransformOut<
   isNonCollapsible,
   campaignId,
   tagId,
+  useDefaultPrompt,
+  overridePrompt,
+  useDefaultTitle,
+  overrideTitle,
+  useDefaultDescription,
+  overrideDescription,
 }: FieldNameToValueMap<typeof calloutFields>): ExternalCalloutData => {
+  const cleanupDescription = (desc: string): string => {
+    // Empty richtext fields have a paragraph tag, so we should check if its empty
+    const noTags = desc.replace(/(<([^>]+)>)/gi, "");
+    return noTags.trim() ? desc : "";
+  };
+
+  const getOptionalFields = () => ({
+    // If we are using the default value, don't include the override field
+    // Otherwise, cleanup the override field and include it
+    ...(!useDefaultPrompt && { overridePrompt: overridePrompt.trim() }),
```

Comment:
I did _not_ know you could do this in objects (although did I know you cannot do this in arrays, hence some confusion 🙃) – TIL, ta!